src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -41,23 +45,47 @@ protected function configure()
             ->setName('translation:update')
             ->setDefinition(array(
                 new InputArgument('locale', InputArgument::REQUIRED, 'The locale'),
-                new InputArgument('bundle', InputArgument::REQUIRED, 'The bundle where to load the messages'),
+                new InputArgument(
+                    'bundle', InputArgument::OPTIONAL ^ InputArgument::IS_ARRAY,
```

Comment:
wouldn't it be better to use 2 options ?

```bundle``` or ```bundles``` and handle them directly as array or not.
if both options are not provided **all** bundles were updated.